matchbox_socket/src/webrtc_socket/native/message_loop.rs

```diff
@@ -416,18 +432,57 @@ async fn create_rtc_peer_connection(
     Ok((connection, trickle))
 }
 
+async fn create_data_channel_pair(
+    connection: &RTCPeerConnection,
+    channel_ready: futures_channel::mpsc::Sender<u8>,
+    peer_id: PeerId,
+    new_peer_tx: UnboundedSender<PeerId>,
+    from_peer_message_tx: UnboundedSender<(PeerId, Channel, Packet)>,
+) -> (Arc<RTCDataChannel>, Arc<RTCDataChannel>) {
+    (
+        create_data_channel(
+            connection,
+            channel_ready.clone(),
+            peer_id.clone(),
+            new_peer_tx.clone(),
+            from_peer_message_tx.clone(),
+            false,
+        )
+        .await,
+        create_data_channel(
+            connection,
+            channel_ready,
+            peer_id,
+            new_peer_tx,
+            from_peer_message_tx,
+            true,
+        )
+        .await,
+    )
+}
+
 async fn create_data_channel(
     connection: &RTCPeerConnection,
     mut channel_ready: futures_channel::mpsc::Sender<u8>,
     peer_id: PeerId,
     mut new_peer_tx: UnboundedSender<PeerId>,
-    from_peer_message_tx: UnboundedSender<(PeerId, Packet)>,
+    from_peer_message_tx: UnboundedSender<(PeerId, Channel, Packet)>,
+    ordered: bool,
 ) -> Arc<RTCDataChannel> {
-    let config = RTCDataChannelInit {
-        ordered: Some(false),
-        max_retransmits: Some(0),
-        negotiated: Some(DATA_CHANNEL_ID),
-        ..Default::default()
+    let config = if ordered {
+        RTCDataChannelInit {
+            ordered: Some(true),
+            max_retransmits: Some(5),
```

Comment:
No it was just random to be honest. According to https://stackoverflow.com/questions/54292824/webrtc-channel-reliability it does not need to be set in case of a reliable data channel so I will simply remove it.